packages/components/src/ui/ariakit-tooltip/types.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import type { TooltipStateProps } from 'ariakit/tooltip';
+
+export type ToolTipProps = Pick< TooltipStateProps, 'placement' > & {
+	/**
+	 * The anchor for the tooltip. Accepts only one child element.
+	 */
+	children: React.ReactElement | string;
+	/**
+	 * The amount of time in milliseconds to wait before showing the tooltip.
+	 *
+	 * @default 700
+	 */
+	delay?: number;
+	/**
+	 * Where the tooltip should be positioned relative to its parent.
+	 *
+	 * @default
+	 */
+	placement?: any; //Placement | undefinded;
```

Comment:
I believe this needs to be `Placement | undefined` (not undefinded 😂 ). I thought I could achieve this with Pick but it's not quite there so another to-do item.